infra/charts/turing/templates/_helpers.tpl

```diff
@@ -186,7 +186,7 @@ MLPConfig:
   MLPEncryptionKey: {{ include "turing.mlp.encryption.key" . | quote }}
 {{ if .Values.tags.mlp }}
   MerlinURL: {{ printf "http://%s:8080/v1" (include "turing.merlin.name" .) }}
-  MLPURL: {{ printf "http://%s:8080/v1" (include "turing.mlp.name" .) }}
+  MLPURL: {{ printf "http://%s:8080" (include "turing.mlp.name" .) }}
```

Comment:
Some docs / sample values need to be updated with the same change:
* https://github.com/caraml-dev/turing/blob/main/README.md?plain=1#L233
* https://github.com/caraml-dev/turing/blob/main/api/.env.development#L30
* https://github.com/caraml-dev/turing/blob/main/api/config-dev.yaml#L91

Reply:
Updated